### PR TITLE
Finished Page UI : Add ui for completed downloads

### PR DESCRIPTION
**Pull request by Nikhil713**

### CODE REVIEW

File: app/src/main/java/com/nevilleantony/prototype/adapters/CompletedListAdapter.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+public class CompletedListAdapter extends RecyclerView.Adapter<CompletedListAdapter.RecyclerViewHolder> {
+
+    private final List<File> downloads;
+    private final Intent downloadIntent;
+    private final Context context;
+
+    public CompletedListAdapter(Context context, List<File> completedList) {
+        this.downloads = completedList;
+        downloadIntent = new Intent(context, DownloadService.class);
```
Comment (kss682) [inline app/src/main/java/com/nevilleantony/prototype/adapters/CompletedListAdapter.java]: Do we need ```downloadIntent``` in completed page, this is for starting service in downloads page, right?

File: app/src/main/java/com/nevilleantony/prototype/adapters/CompletedListAdapter.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    public RecyclerViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        LayoutInflater inflater = LayoutInflater.from(parent.getContext());
+        View view = inflater.inflate(R.layout.fragment_completed_list_row, parent, false);
+        return new RecyclerViewHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull RecyclerViewHolder holder, int position) {
+        File fileDownload = downloads.get(position);
+        holder.downloadsName.setText(fileDownload.getName());
```
Comment (NevilleAntony98) [inline app/src/main/java/com/nevilleantony/prototype/adapters/CompletedListAdapter.java]: Here you also need to set an onClick listener on the share button, which creates an intent for ShareActivity.class. You should also add the absolute path of the file as an extra in the intent and call startActivity with it.

File: app/src/main/res/layout/fragment_completed_list_row.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            android:orientation="horizontal"
+            android:padding="10dp">
+
+            <TextView
+                android:id="@+id/completed_filename"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_weight="9"
+                android:paddingVertical="15dp"
+                android:text="@string/completed_filename_text"
```
Comment (NevilleAntony98) [inline app/src/main/res/layout/fragment_completed_list_row.xml]: This line can be removed

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -51,6 +51,9 @@
 
     <string name="downloads_percent_initial">00</string>
 
+    <string name="share_button_text">Share</string>
+    <string name="completed_filename_text">TextView</string>
```
Comment (NevilleAntony98) [inline app/src/main/res/values/strings.xml]: This line can be removed

**Review by NevilleAntony98 — CHANGES_REQUESTED** [review]

**Review by NevilleAntony98 — APPROVED** [review]

**Pull request merged**